CONTRIBUTING.md

```diff
@@ -66,6 +66,8 @@ We provide a script that can automatically create all the necessary files for yo
 
 - Add all your component styles in `src/[component-name]/[component-name].scss` file.
   - Break variables and mixins into separate files where possible.
+  - Use existing Sass variables for spacing (margin, padding, sizing, etc).
+  - Use [`themed`](https://github.com/iTwin/iTwinUI/blob/main/src/style/theme.scss#L430) mixin wherever color is involved.
```

Comment:
Maybe add an example how to use it.

Reply:
themed mixin has example comment above it

Reply:
There is one if you click the link.

Reply:
Improved the example